src/main/java/com/alipay/oceanbase/rpc/ObTableClient.java

```diff
@@ -1389,7 +1427,7 @@ public ObPair<Long, ObTableParam> getTable(String tableName, List<ObNewRange> ke
     /**
      * get addr by pardId
      * @param tableName table want to get
-     * @param partId partId where table located
+     * @param partId partId where table located (should be partIdx)
```

Comment:
is it more understandable to rename the `partId` parameter to `partIdx`

Reply:
Actually, it is partition id in 3.x, which is different from partIdx and tablet id in 4.x. So I have commented on it again.